<ui version="4.0" >
 <class>Material</class>
 <widget class="QWidget" name="Material" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Material</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Peso Específico</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Código</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLineEdit" name="codigoEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="frame" >
          <bool>true</bool>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="nomeEdit" >
       <property name="maxLength" >
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="submitButton" >
         <property name="text" >
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>&amp;Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2" >
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2" >
      <widget class="QDoubleSpinBox" name="densidadeEdit" >
       <property name="maximumSize" >
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals" >
        <number>4</number>
       </property>
       <property name="maximum" >
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.500000000000000</double>
       </property>
       <property name="value" >
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
